pkg/ocmagenthandler/ocmagenthandler_secret.go

```diff
@@ -36,8 +37,10 @@ func (o *ocmAgentHandler) ensureAccessTokenSecret(ocmAgent ocmagentv1alpha1.OcmA
 
 	clusterPullSecret, err := o.fetchAccessTokenPullSecret()
 	if err != nil {
+		localmetrics.UpdateMetricPullSecretInvalid(ocmAgent.Name)
```

Comment:
seems the metric here indicates only the format of the pull secret, not sure if we can valid the contains.
Out of scope but right my thoughts down here.
Maybe move this to the ocm-agent, when the ocm request return code is 40x in the webhookreceiver.

Reply:
I'd have to review the PR I made to the OCM SDK, but I think it (the OCM SDK) might will raise an error if a malformed token is supplied. In which case yeah we can have ocm-agent produce more detailed metrics there.